CHANGELOG.md

```diff
@@ -22,6 +22,11 @@
 * Provides admin support for Cluster-aware version of Hoist.
 * `Store` now supports multiple `summaryRecords`.
 
+### ⚙️ Technical
+
+* `ping` call removed from `FetchService.ts`.  `ping` check now suggested  in alert msg
+  if server unavailable error occurs during app initialization.
```

Comment:
Need to fixup this message - we can clean it up before release as I am sure we'll loop over the CL

Reply:
done in new PR: https://github.com/xh/hoist-react/pull/3639